### PR TITLE
Increase timeouts to avoid intermittent test failures

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

### CODE REVIEW

**Comment by kevinearls** [comment]
@jpkrohling  @pavolloffay  @objectiser  Please review.  This is something we've seen twice recently on both a PR (https://github.com/jaegertracing/jaeger-operator/pull/856/checks?check_run_id=394853911) and on internal CI.

File: test/e2e/spark_dependencies_test.go [hunk 1]
```diff
@@ -42,12 +43,12 @@ func sparkTest(t *testing.T, f *framework.Framework, testCtx *framework.TestCtx,
 	}
 	defer undeployJaegerInstance(j)
 
-	err = WaitForCronJob(t, f.KubeClient, namespace, fmt.Sprintf("%s-spark-dependencies", name), retryInterval, timeout)
+	err = WaitForCronJob(t, f.KubeClient, namespace, fmt.Sprintf("%s-spark-dependencies", name), retryInterval, timeout+timeout+1*time.Minute)
```
Comment (jpkrohling) [inline test/e2e/spark_dependencies_test.go]: Can you change it to something easier to read? Like, `5 * time.Minute` (or whatever the calculation yields)

Comment (kevinearls) [reply test/e2e/spark_dependencies_test.go]: @jpkrohling I would but then @objectiser might object.  He suggested I do it this way for this PR https://github.com/jaegertracing/jaeger-operator/pull/847  :smirk:

Comment (objectiser) [reply test/e2e/spark_dependencies_test.go]: I thought it best to be relative to the global `timeout` value - however in this PR it looks like `timeout` has been added two times? Not sure this is necessary.

**Pull request merged**